### PR TITLE
New event

### PR DESCRIPTION
**Pull request by bandyto**

### CODE REVIEW

File: app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/Event.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package pro.itshark.moneysplitter.presentation.newevent
+
+import android.databinding.BaseObservable
+import android.databinding.Bindable
+import pro.itshark.moneysplitter.BR
+
+data class Event(
```
Comment (BlaizMiko) [inline app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/Event.kt]: `EventModel`

File: app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/Event.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        private var _title: String = "",
```
Comment (BlaizMiko) [inline app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/Event.kt]: For what prefix `_`

File: app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/Event.kt [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        private var _description: String = "",
```
Comment (BlaizMiko) [inline app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/Event.kt]: change "" to constants

File: app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/NewEventActivity.kt [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+import android.widget.Toast
+import dagger.android.AndroidInjection
+import pro.itshark.moneysplitter.R
+import pro.itshark.moneysplitter.databinding.ActivityAddEventBinding
+import javax.inject.Inject
+
+class NewEventActivity : AppCompatActivity() {
+
+    companion object {
+        fun start(context: Context) = context.startActivity(Intent(context, NewEventActivity::class.java))
```
Comment (BlaizMiko) [inline app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/NewEventActivity.kt]: Why do u use `.java`. Could we avoid this?

File: app/src/main/java/pro/itshark/moneysplitter/presentation/userProfile/UserProfileActivity.kt [hunk 5]
```diff
@@ -20,16 +22,18 @@ class UserProfileActivity : AppCompatActivity() {
     }
 
     @Inject
-    lateinit var viewModel: UserProfileViewModel
+    lateinit var viewMOdelFactory: ViewModelProvider.Factory
```
Comment (BlaizMiko) [inline app/src/main/java/pro/itshark/moneysplitter/presentation/userProfile/UserProfileActivity.kt]: viewM`O`delFactory

File: app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/NewEventActivity.kt (hunk 4 above)
Comment (bandyto) [reply app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/NewEventActivity.kt]: done

**Pull request merged**